docs/file-events.yml

```diff
@@ -2335,7 +2342,6 @@ components:
     SearchRequest:
       title: SearchRequest
       required:
-        - defaultSortKey
```

Comment:
making this required was a breaking change from the requests we're already sending